{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## For the Submitter
```

Comment:
I think there should be a link to the wiki documentation about the PR review process. I'm not sure where it should go as it should be there both for the developer and for the reviewers. Maybe it should be in the PR Cover Letter section as that is the first section shown to both developers and reviewers.